communications/rfc.rst

```diff
@@ -177,6 +177,12 @@ If an RFC includes API changes, including deprecations, or updates to a data mod
 
 In all of these cases, the RFC is marked as **Flagged** to indicate that it is being considered by the DMCCB.
 
+As a result of the discussion on a **Flagged** RFC, the DMCCB may decide to approve it and move forward with the proposal.
+In this case, the RFC status is set to **Board Recommended**.
+Once one or more triggering issues have been opened on the RFC, the assignee has to change the status to **Adopted**.
```

Comment:
Maybe instead say:

> The assignee can then attach the triggering issues and adopt the RFC following the standard process.

?

Reply:
I am not convinced about the use of the verb "can".

Reply:
Do you prefer `should`?  I was trying not to imply we were forcing people to adopt the RFC.

Reply:
OK, It can be used "can" as you suggest, and add at the end ".... standard process, or reopen it setting its status back to **Flagged**."